introduction.md

```diff
@@ -1,2 +1,3 @@
 Hello everybody! My name is Pedro Fernandez, and love to work as a tester on agile environments. I am learning git :-)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?